tests/evaluator/haspi/test_eb.py

```diff
@@ -293,7 +292,7 @@ def test_env_compress_basilar_membrane():
         fsamp=24000,
         level1=140,
         small=1e-30,
-        threshold_high=120,
+        threshold_high=100,
```

Comment:
Because this parameter had no impact in the test as the `threshold_high` was then hardcoded to 100 in the `env_compress_basilar_membrane` function. I change the test to 100 to preserve the previous results